fusion-plugin-i18n/src/__tests__/index.node.js

```diff
@@ -168,3 +168,68 @@ test('non matched route', async t => {
   t.notok(ctx.body, 'does not set ctx.body');
   t.end();
 });
+
+test('matchesOrder matches positionally', async t => {
```

Comment:
Purpose of tests in this file:
- static analysis output is used accurately in a number of cases to match translations